apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap

```diff
@@ -1968,6 +1986,38 @@ expression: "&schema"
       ],
       "type": "object"
     },
+    "Event_for_RouterSelector": {
+      "oneOf": [
+        {
+          "description": "For every chunk sent through http multipart connection",
```

Comment:
This throws me off a bit, intuitively I read that as multipart file upload chunks. Also does this actually apply only to http multipart /responses/ ?

Reply:
Hmm not really, I don't know how to explain it better because it's highly related to our internals. It's basically event coming in the supergraph response stream. Maybe a better description could be `For every supergraph response payload (including subscription events and defer events)` ?

Reply:
cc @shorgi WDYT?

I'm going to preapprove the PR and let Edward give his opinion / suggest something :D

Reply:
@bnjjj I like your updated suggestion.

Reply:
I updated with my suggestion and enable auto merge in order to have this in the next release. Once we have feedback from @shorgi I'll open a new PR to change the description